async port scaner.py

```diff
@@ -0,0 +1,31 @@
+import socket
+import asyncio
+import time
+t=time.time()
+port=0
+async def scan(i: int):
+    """
+    check if a port for a socket is valid and save it as global i if True
+    :param i: the port for the socket to try and connect with
+    """
+    try:
+        global port
+        await asyncio.open_connection("127.0.0.1",i)
+        print(f"{i} found")
+        port=i
+    except:
+        print(f"{i} not found")
```

Comment:
No need to print ports that are not found. This can be inherited by just knowing what was actually found.

Reply:
If there is no need for the `except` block then you can use `surpress`.